src/qps/interface/qps.c

```diff
@@ -1160,75 +1163,46 @@ PetscErrorCode QPSMonitor(QPS qps, PetscInt it, PetscReal rnorm)
 
    Input Parameters:
 +  qps - iterative context obtained from QPSCreate()
-.  monitor - pointer to function (if this is NULL, it turns off monitoring
-.  mctx    - [optional] context for private data for the
-              monitor routine (use NULL if no context is desired)
--  monitordestroy - [optional] routine that frees monitor context
-                    (may be NULL)
+.  monitor - pointer to function
+.  mctx    - [optional] context for private data for the monitor routine (use NULL if no context is desired)
+-  monitordestroy - [optional] routine that frees monitor context (may be NULL)
 
    Calling Sequence of monitor:
-$     monitor (QPS qps, int it, PetscReal rnorm, void *mctx)
+$     monitor (QPS qps, PetscInt it, PetscReal rnorm, void *mctx)
 
 +  qps - iterative context obtained from QPSCreate()
 .  it - iteration number
-.  rnorm - (estimated) 2-norm of (preconditioned) residual
+.  rnorm - estimated 2-norm of the residual
 -  mctx  - optional monitoring context, as set by QPSMonitorSet()
 
    Options Database Keys:
 +  -qps_monitor - sets QPSMonitorDefault()
-.  -qps_monitor_true_residual    - sets QPSMonitorTrueResidualNorm()
-.  -qps_monitor_max    - sets QPSMonitorTrueResidualMaxNorm()
-.  -qps_monitor_lg_residualnorm    - sets line graph monitor,
-                             uses QPSMonitorLGResidualNormCreate()
-.  -qps_monitor_lg_true_residualnorm   - sets line graph monitor,
-                             uses KSPMonitorLGResidualNormCreate()
-.  -qps_monitor_singular_value    - sets QPSMonitorSingularValue()
--  -qps_monitor_cancel - cancels all monitors that have
-         been hardwired into a code by
-         calls to QPSMonitorSet(), but
-         does not cancel those set via
-         the options database.
+-  -qps_monitor_cancel - Cancels all monitors set from options or with QPSMonitorSet().
```

Comment:
```suggestion
-  -qps_monitor_cancel - cancels all monitors that have
         been hardwired into a code by
         calls to QPSMonitorSet(), but
         does not cancel those set via
         the options database.
```
see above

Reply:
OK, can we just reduce it to two lines? This looks terrible :-D
```suggestion
-  -qps_monitor_cancel - cancels all monitors that have been hardwired into a code by
    calls to QPSMonitorSet(), but does not cancel those set via the options database.
```